### PR TITLE
Aws fixes

### PR DESCRIPTION
**Pull request by egopher**

### CODE REVIEW

File: pkg/model/kube.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ type Kube struct {
 // Auth holds all possible auth parameters.
 type Auth struct {
 	Username string `json:"username"`
-	Token    string `json:"token"`
```
[the remark this answers is not in the document]
Comment (egopher) [reply pkg/model/kube.go]: Fixed

File: pkg/workflows/steps/amazon/delete_cluster_test.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+type FakeEC2KDeleteCluster struct {
+	ec2iface.EC2API
+
+	instancesOutput  *ec2.DescribeInstancesOutput
+	terminatedOutput *ec2.TerminateInstancesOutput
+
+	err error
+}
+
```
[the remark this answers is not in the document]
Comment (egopher) [reply pkg/workflows/steps/amazon/delete_cluster_test.go]: Will add later.

**Comment by eldridgeh** [comment]
After creating a (1)4gb master with (1)4gb node, the cluster spins up in AWS, however, I imediantly get the below:
`ERRO[2333] helm: list releases: aws cluster: list releases: get tiller pod: Get http://54.183.123.43:8080/api/v1/pods?labelSelector=app%3Dhelm%2Cname%3Dtiller: dial tcp 54.183.123.43:8080: i/o timeout 
ERRO[2344] helm: list releases: aws cluster: get kube: storage: get: entity not found 
ERRO[2354] helm: list releases: aws cluster: list releases: get tiller pod: Get http://54.183.123.43:8080/api/v1/pods?labelSelector=app%3Dhelm%2Cname%3Dtiller: dial tcp 54.183.123.43:8080: i/o timeout 
ERRO[2354] helm: list releases: aws cluster: get kube: storage: get: entity not found `

Also looks like prometheus did not install. I was successfully able to delete the cluster with the UI.

**Comment by eldridgeh** [comment]
@egopher 
It doesn't look like security groups is not being cleaned up on deletion of a cluster.

**Pull request merged**